cmd/clusterctl/client/cluster/mover.go

```diff
@@ -565,6 +571,11 @@ func (o *objectMover) deleteGroup(group moveGroup) error {
 	for i := range group {
 		nodeToDelete := group[i]
 
+		// Don't delete cluster-wide nodes
```

Comment:
Added support for not deleting global resources

Reply:
nit
create `nodeToDelete.IsGlobal`

Reply:
Also, wondering if we should track a flag at Type level for identifying global resources vs a namespaced resource (instead of looking at the absence of namespace value).

Reply:
updated with isglobal bool. gonna resolve these comments.